window_manager/BUILD

```diff
@@ -37,6 +38,7 @@ cc_binary(
     deps = [
         ":rcbe_window_manager",
         "//common:engine_json_utils",
+        "//datamodel:engine_datamodel_math",
```

Comment:
Why?

Reply:
Same debug build problem.

Reply:
Yeah, but this one is much less logical then the previous one.